api.planx.uk/hasura/metadata/index.ts

```diff
@@ -25,13 +25,18 @@ type RequiredScheduledEventArgs = Pick<
   "webhook" | "schedule_at" | "comment" | "payload"
 >;
 
+export interface ScheduledEventResponse {
```

Comment:
Docs: https://hasura.io/docs/latest/api-reference/metadata-api/scheduled-triggers/#metadata-create-scheduled-event

Reply:
Typing this cascaded across a few functions and tests which is a bit noisy for a PR, but quite nice in practice.